tests/dbus/test_dbus_client.cpp

```diff
@@ -267,6 +267,27 @@ void CheckTelemetryData(ThreadApiDBus *aApi)
     TEST_ASSERT(telemetryData.wpan_stats().phy_tx() > 0);
     TEST_ASSERT(telemetryData.wpan_stats().phy_rx() > 0);
     TEST_ASSERT(telemetryData.wpan_stats().ip_tx_success() > 0);
+    TEST_ASSERT(telemetryData.wpan_topo_full().rloc16() > 0);
+    TEST_ASSERT(telemetryData.wpan_topo_full().network_data().size() > 0);
+    TEST_ASSERT(telemetryData.wpan_topo_full().partition_id() > 0);
+    TEST_ASSERT(telemetryData.wpan_topo_full().extended_pan_id() > 0);
+    TEST_ASSERT(telemetryData.topo_entries_size() == 1);
+    TEST_ASSERT(telemetryData.topo_entries(0).rloc16() > 0);
+    TEST_ASSERT(telemetryData.wpan_border_router().border_routing_counters().rs_tx_failure() == 0);
+#if OTBR_ENABLE_SRP_ADVERTISING_PROXY
+    TEST_ASSERT(telemetryData.wpan_border_router().srp_server().state() ==
+                threadnetwork::TelemetryData::SRP_SERVER_STATE_RUNNING);
+#endif
+#if OTBR_ENABLE_DNSSD_DISCOVERY_PROXY
+    TEST_ASSERT(telemetryData.wpan_border_router().dns_server().response_counters().server_failure_count() == 0);
+#endif
+    TEST_ASSERT(telemetryData.wpan_border_router().mdns().service_registration_responses().success_count() > 0);
+#if OTBR_ENABLE_NAT64
+    TEST_ASSERT(telemetryData.wpan_border_router().nat64_state().prefix_manager_state() ==
+                threadnetwork::TelemetryData::NAT64_STATE_NOT_RUNNING);
+#endif
+    TEST_ASSERT(telemetryData.wpan_rcp().rcp_interface_statistics().transferred_frames_count() > 0);
+    TEST_ASSERT(telemetryData.coex_metrics().count_tx_request() > 0);
 }
```

Comment:
It may be hard to verify these metrics in this test due to the limitation of the test environment. It may be better to verify them in python BR tests.

Reply:
SG. Will be added in a later CL.